.pre-commit-config.yaml

```diff
@@ -1,12 +1,12 @@
-repos:
```

Comment:
Are the changes here intentional?

Reply:
Nope. After I ran the pre-commit test, and everything was fine, I was asked to stage this file for commit as well, and so I did.

Reply:
Hmm, not sure what happened there. Would you mind reverting this change then?